### PR TITLE
fix: delete orphan charts on dashboard cancel

### PR DESCRIPTION
**Pull request by ZeRego**
Closes: [#6591](https://github.com/lightdash/lightdash/issues/6591) 

### Description:

https://github.com/lightdash/lightdash/assets/9117144/f3119317-e42f-4dd5-8929-a9a0583cb382

### CODE REVIEW

**Review by rephus — APPROVED** [review]
LGTM, nice demo video

File: packages/frontend/src/pages/Dashboard.tsx [hunk 1]
```diff
@@ -294,6 +295,32 @@ const Dashboard: FC = () => {
...
+                const isChartNew =
+                    (dashboard?.tiles || []).find(
+                        (t) =>
+                            isDashboardChartTileType(t) &&
+                            t.properties.savedChartUuid ===
+                                tile.properties.savedChartUuid,
+                    ) === undefined;
+
+                if (isChartNew) {
+                    deleteSavedQuery(tile.properties.savedChartUuid).catch(
```
[the remark this answers is not in the document]
Comment (ZeRego) [reply packages/frontend/src/pages/Dashboard.tsx:315]: yes, because we don't need to handle the state ( loading, success, etc )

**Comment by rephus** [comment]
Retriggering render

**Comment by rephus** [comment]
:sheep:  it !

**Pull request merged**